code/modules/goals/definitions/personal_achievement_specific_object.dm

```diff
@@ -29,6 +29,11 @@
 
 /datum/goal/achievement/specific_object/food/New()
 	possible_objects = subtypesof(/obj/item/chems/food/snacks)
+	blacklisted_objects = blacklisted_objects || list()
+	blacklisted_objects |= subtypesof(/obj/item/chems/food/snacks/amanita_pie)
```

Comment:
Are there actually subtypes of this, or did you mean `typesof`?

Reply:
I did mean typesof, whoops